pyServer/manifests/windows/dump

```diff
@@ -0,0 +1,2 @@
+echo,### Memory Dump ###
+copy,/Windows/memory.dmp
```

Comment:
Could you remove the memory dump collection from this PR since that's a separate effort?

Reply:
Done